src/core/ddsi/src/q_radmin.c

```diff
@@ -620,7 +626,7 @@ void nn_rmsg_commit (struct nn_rmsg *rmsg)
   ASSERT_RBUFPOOL_OWNER (chunk->rbuf->rbufpool);
   ASSERT_RMSG_UNCOMMITTED (rmsg);
   assert (chunk->u.size <= chunk->rbuf->max_rmsg_size);
-  assert ((chunk->u.size % 8) == 0);
+  assert ((chunk->u.size % (sizeof(void *) > 8 ? sizeof(void *) : 8)) == 0);
```

Comment:
Maybe extract to a constant `ALIGNOF_RMSG`

Reply:
just pushed a change.  see if it builds...